src/omnicore/log.cpp

```diff
@@ -236,7 +236,7 @@ void InitDebugLogLevels()
     for (const auto& d : debugLevels) {
         auto none = d == "none";
         auto all = d == "all";
-#define ENABLE_LOG(x) msc_debug_##x = none ? false : (all || d == #x)
+#define ENABLE_LOG(x) msc_debug_##x = none ? false : (all || d == #x || msc_debug_##x)
```

Comment:
We could do it smart
```
#define ENABLE_LOG(x) msc_debug_##x |= none ? false : (all || d == #x)
```

Reply:
> We could do it smart
> 
> ```
> #define ENABLE_LOG(x) msc_debug_##x |= none ? false : (all || d == #x)
> ```

Not the same, it will not work if we want to switch off all the flags when setting "none"